wd_core.au3

```diff
@@ -1285,7 +1285,7 @@ Func _WD_Startup()
 
 		__WD_ConsoleWrite($sFuncName & ": OS:" & @TAB & @OSVersion & " " & @OSType & " " & @OSBuild & " " & @OSServicePack)
 		__WD_ConsoleWrite($sFuncName & ": AutoIt:" & @TAB & @AutoItVersion)
-		__WD_ConsoleWrite($sFuncName & ": au3WD UDF:" & @TAB & $__WDVERSION & $sUpdate)
+		__WD_ConsoleWrite($sFuncName & ": Webdriver UDF:" & @TAB & $__WDVERSION & $sUpdate)

```

Comment:
au3WebDriver

Reply:
Nope. As I previously stated, that is the name of the repo. I still refer to the UDF as "Webdriver", but changing the repo name avoids confusion with the W3C Webdriver repo.

Reply:
Ok